articles/quickstart/webapp/aspnet-core-2/index.yml

```diff
@@ -12,7 +12,7 @@ snippets:
   dependencies: server-platforms/aspnet-core/dependencies
 alias:
   - aspnetcore
-  - aspnetcore-3.1
+  - aspnetcore-2.0
```

Comment:
Is the alias `aspnetcore` correct? And should we use `aspnetcore-2.1` instead of `2.0` ?

Reply:
Corrected these